crates/gitbutler-operating-modes/src/lib.rs

```diff
@@ -1,38 +1,113 @@
+use std::{fs, path::PathBuf};
+
 use anyhow::{bail, Context, Result};
 use gitbutler_command_context::CommandContext;
+use serde::{Deserialize, Serialize};
+
+pub const INTEGRATION_BRANCH_REF: &str = "refs/heads/gitbutler/integration";
+pub const EDIT_BRANCH_REF: &str = "refs/heads/gitbutler/edit";
```

Comment:
what is this reference for? Also, should this be excluded when listing branches?

Reply:
You are on `gitbutler/edit` while in edit mode. It should be excluded from listing branches